js/parts/Navigator.js

```diff
@@ -1400,7 +1400,7 @@ Navigator.prototype = {
 			xAxisIndex = chart.xAxis.length,
 			yAxisIndex = chart.yAxis.length,
 			baseXaxis = baseSeries && baseSeries[0] && baseSeries[0].xAxis ||
-				chart.xAxis[0];
+				chart.xAxis[0] || { options: {} };
```

Comment:
This can be replaced with `{ options: H.Axis.prototype.defaultBottomAxisOptions }`.